util/util.go

```diff
@@ -9,6 +9,10 @@ import (
 	"strings"
 )
 
+const (
+	OSRELEASE_PATH = "/etc/os-release"
```

Comment:
Agreed, the naming is confusing - having worked on the supervisor side too long has made me relate the word "resin" to "The Resin API" too much. That's why it made sense to me, but I see why it wouldn't for other people.

And yeah, "util" is generally a horrible name, but for me it's a way to separate shared useful functionality while we discover the things that will be package-able for the bigger picture.